### PR TITLE
Fix typo-s in robot class

### PR DESCRIPTION
**Pull request by sdimkov**

### CODE REVIEW

File: src/robot.coffee [hunk 1]
```diff
@@ -392,7 +392,6 @@ class Robot
   # Returns nothing.
   parseHelp: (path) ->
     @logger.debug "Parsing help for #{path}"
-    scriptName = Path.basename(path).replace /\.(coffee|js)$/, ''
```
Comment (michaelansel) [inline src/robot.coffee]: #917 actually depends on this being present. In addition, I'd much prefer to have test cases in place before modifying this code.

**Comment by michaelansel** [comment]
Please remove the code change per my comments and I'll merge

**Comment by sdimkov** [comment]
I've removed the commit. 

Are you sure #917 will get merged in the near future? It's open for the last 3 months

I agree with you in general about the importance of tests and coverage.Yet in this particular case that argument makes very little sense.

**Comment by technicalpickles** [comment]
> Are you sure #917 will get merged in the near future?

I'm not really sure of it. I left my feedback down in https://github.com/github/hubot/pull/917#issuecomment-113532776 there.

**Pull request merged**